src/components/Footer.vue

```diff
@@ -45,4 +45,9 @@
 export default {
   name: 'Footer',
 }
-</script>
\ No newline at end of file
+</script>
+<style lang="scss">
+h5{
+    font-weight: 700 !important;
```

Comment:
Don't use `important!` at least it is strictly necessary, in this case, if you use the `scoped` attribute in the `style` this won't be necessary.
```suggestion
    font-weight: 700;
```